scripts/qtsupport.py

```diff
@@ -20,11 +20,19 @@
     You should have received a copy of the GNU General Public License
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 '''
-import sys, math, re, logging, Queue
+import sys, math, re, logging
+import sys
+is_py2 = sys.version[0] == '2'
+if is_py2:
+    import Queue as queue
```

Comment:
This shouldn't be needed. Just do `import queue as Queue` which should work for both py2 and py3.

Reply:
Thanks for the tip; realised it's not needed, see https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/248/commits/61e11cc5b27fac1b7c3198f466f159f4f681148b